docs/src/dev-guide/design-kv-ir-streams/background.md

```
@@ -0,0 +1,316 @@
+    "themeVariables": {
+      "primaryColor": "#0066cc",
+      "primaryTextColor": "#fff",
+      "primaryBorderColor": "transparent",
+      "lineColor": "#007fff",
+      "secondaryColor": "#007fff",
+      "tertiaryColor": "#fff"
+    }
+  }
+}%%
+flowchart LR
+  rootObj("root: <span style='color: #97ff00'>Object</span>")
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: span

(MD033, no-inline-html)